.kuzzlerc.sample

```diff
@@ -547,14 +547,14 @@
         "roles": {
           "properties": {
             "controllers": {
-              "dynamic": "false,
+              "dynamic": "false",
```

Comment:
shouldnt be a boolean?

Reply:
No: values are "true", "false" and "strict"

Reply:
No, Elasticsearch expect only string value here since it could be either `"false"`, `"true"` or `"strict"`

Reply:
No, Elasticsearch expect only string value here since it could be either `"false"`, `"true"` or `"strict"`